lib/collection/src/collection.rs

```diff
@@ -902,6 +903,7 @@ impl Collection {
             ids: search_result.iter().map(|x| x.id).collect(),
             with_payload,
             with_vector,
+            read_consistency: ReadConsistency::Factor(1), // TODO: Which `ReadConsistency` should be used here!?
```

Comment:
See the `TODO`.

Seems like we should simply propagate read consistency (from the `search_batch` method) used by respective search request in the batch, but I'd better clarify just in case.

Reply:
Ditto.